implementations/micrometer-registry-statsd/src/test/java/io/micrometer/statsd/StatsdMeterRegistryTest.java

```diff
@@ -57,6 +57,7 @@ class StatsdMeterRegistryTest {
 
     private MockClock clock = new MockClock();
 
+    @SuppressWarnings("NullAway.Init")
```

Comment:
Can we create an issue to fix this or fix it here?
Since every test case creates it's own registry anyways, we can remove this field with `cleanUp` and replace `registry =` to `StatsdMeterRegistry registry =`.

I guess the same for `StatsdMeterRegistryPublishTest` (`meterRegistry`, `server`, `serverLatch`).

Reply:
I'd vote for a separate issue so it isn't buried in all these changes.

Reply:
Done in https://github.com/micrometer-metrics/micrometer/issues/6284